dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -127,14 +125,15 @@ def __alb_schema() -> dict:
         }
 
     @staticmethod
-    def __codebase_pipelines_schema() -> list[dict]:
-        return [
-            {
-                "name": str,
```

Comment:
How is the pipeline named now?

Reply:
It's the key in the map. Same as environment pipelines.